src/hyperfine/shell.rs

```diff
@@ -62,7 +62,7 @@ fn run_shell_command(command: &str) -> io::Result<std::process::ExitStatus> {
 
 /// Run a Windows shell command using cmd.exe
 #[cfg(windows)]
-fn run_shell_command(command: &str) -> io::Result<std::process::Child> {
+fn run_shell_command(_: &str, command: &str) -> io::Result<std::process::Child> {
     Command::new("cmd")
```

Comment:
Why should this be disabled for Windows? Granted, it would only work if the shell accepts `/C`.